Assets/Scripts/Framework/Gameplay/Interactions/Interactable.cs

```diff
@@ -39,9 +39,10 @@ public class Interactable : NetworkBehaviour
         /// A function that calls a RPC to the server on this Interactable.
         /// </summary>
         /// <param name="interactor">The interactor that this request originates from.</param>
-        [ServerRpc(RequireOwnership = false)]
+        [ServerRpc(RequireOwnership = false), Server]
```

Comment:
This one is not needed because it already has a `[ServerRpc]` attribute.
```suggestion
        [ServerRpc(RequireOwnership = false)]
```